shared_model/validators/field_validator.hpp

```diff
@@ -29,17 +29,26 @@
 namespace shared_model {
   namespace validation {
 
+    // default value for future_gap field of FieldValidator
+    constexpr auto default_future_gap =
```

Comment:
Don't agree with you. Validator checks field for being correct **in** the system, not based on it's own mind

Reply:
Please clarify "it's own mind". This variable is not used outside the class.

Reply:
It can be useful for users of that API to get this number

Reply:
@l4l User can use FieldValidator::default_future_gap, it will not pollute namespace and be accessible to any user who wants to use constraints on the system.

Reply:
Don't really like it but let it be